server/views/pages/create/initialMeetingPerson.njk

```diff
@@ -18,14 +19,19 @@
                 {{ govukInput({
                     id: "contactName",
                     name: "contactName",
+                    label: {
+                        text: "contact name",
+                        classes: "govuk-visually-hidden"
+                    },
```

Comment:
Is there a reason for hiding the label here?

Reply:
Label is hidden for accessibility - Axe test failed. We dont want to see it but screen reader will see it